app/controllers/task_lists_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+class TaskListsController < ApplicationController
+  before_action :require_login
+
+  before_action :find_task_list, only: %i[show edit update destroy]
+
+  def index
+    @task_lists = TaskList.where(hidden: false).order('created_at ASC')
```

Comment:
Nit: I think you can skip the `ASC` keyword, since the `ORDER BY` keyword sorts the records in ascending order by default.